OpenSim/Simulation/Model/Model.cpp

```diff
@@ -794,7 +782,7 @@ void Model::extendConnectToModel(Model &model)
             Body* child = static_cast<Body*>(mob.getOutboardBodyRef());
             cout << "Body '" << child->getName() << "' not connected by a Joint.\n"
                 << "A FreeJoint will be added to connect it to ground." << endl;
-            Body* ground = static_cast<Body*>(mob.getInboardBodyRef());
+            Ground* ground = static_cast<Ground*>(mob.getInboardBodyRef());
```

Comment:
Does this change make L788 redundant?

Reply:
L788 verifies that the *ground* that was used to build the graph is also the `Ground` of the the `Model`. It likely isn't necessary to check anymore. Is your preference to remove?

Reply:
That is correct.